examples/community/stable_diffusion_controlnet_inpaint.py

```diff
@@ -0,0 +1,1072 @@
+import inspect
```

Comment:
Can we have file headers where we mention the original authors?
We can also mention them in the docstrings.

Reply:
done!